lms/envs/aws.py

```diff
@@ -490,3 +490,6 @@
 
 ##### METRICS DATA SOURCE #####
 MAX_ENROLLEES_FOR_METRICS_USING_DB = ENV_TOKENS.get('MAX_ENROLLEES_FOR_METRICS_USING_DB', MAX_ENROLLEES_FOR_METRICS_USING_DB)
+
+####################### Forum mongo credentials #######################
```

Comment:
Heh, I know you're just following along with the existing convention here with the big bunch of leading `#`'s, but it actually turns out to be a Pylint violation. Naturally, we shouldn't mess with the existing lines, but how do you feel about simplifying this to:

```python
# Forum mongo credentials
```